docs/src/main/sphinx/sql/show-functions.md

```diff
@@ -3,7 +3,7 @@
 ## Synopsis
 
 ```text
-SHOW FUNCTIONS [ FROM schema ] [ LIKE pattern ]
+SHOW FUNCTIONS [ FROM | IN schema ] [ LIKE pattern ]
```

Comment:
We don't document the `IN` keyword for `SHOW TABLES`, etc. It's more of a hidden feature for user convenience / compatibility.

Reply:
ok